### PR TITLE
trivial: Use g_test_build_filename() to avoid reinventing the wheel

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by hughsie** [comment]
@jtojnar

File: libjcat/jcat-self-test.c [hunk 1]
```diff
@@ -751,6 +733,12 @@ jcat_context_verify_item_csum_func (void)
 int
 main (int argc, char **argv)
 {
+	/* installed tests data files are not in libexec */
```
Comment (smcv) [inline libjcat/jcat-self-test.c]: Is there any particular reason why they aren't? In other GLib'ish packages with installed-tests, it's conventional to use `/usr/share/installed-tests/PACKAGE` for only the `.test` metadata, and put all the test executables, helper programs and their associated data in `/usr/libexec/installed-tests/PACKAGE`.

But if you prefer to use `/usr/share/installed-tests/PACKAGE` to store the test data too, then this change seems correct.

Comment (hughsie) [reply libjcat/jcat-self-test.c]: > In other GLib'ish packages

There's no reason, other than I figured it was weird to put noarch files in an arch-specific directory.

Comment (hughsie) [reply libjcat/jcat-self-test.c]: All fixed, thanks for the review.

**Pull request merged**